src/main/java/com/parasoft/demoapp/config/rabbitmq/RabbitMQInventoryResponseQueueListener.java

```diff
@@ -43,6 +47,11 @@ public void onMessage(Message message) {
             return;
         }
 
-        orderMQService.sendToInventoryRequestDestination(messageToReply);
+        String replyTo = message.getMessageProperties().getReplyTo();
+        if(replyTo != null && replyTo.startsWith(AMQ_RABBITMQ_REPLY_TO)) {
```

Comment:
To make PDA support <a href="https://www.rabbitmq.com/direct-reply-to.html">Direct Reply-to</a> for RabbitMQ, and I have tested it sending and consuming message by RabbitMQ extension in SOAtest.